pictures.md

```diff
@@ -0,0 +1,145 @@
+# Picture Quality Considerations
+
+The subject should not be degraded to the point of being obscured in noise,
+or overly smoothed out, removing the detail of the subject.
+
+The background should not be degraded to the point of being distracting in the context of the subject.
+
+Recommended to avoid:
+
+- Heavy visual degradation. Drop the format if it cannot meet expected quality under 200K at 1080p.
+- Any unexpected smoothing of lines
+- Any unexpected blurring between strong colours (i.e. reddish-pink & black)
+- Any strong colour blending that shouldn't exist
+- Any muteness in colour.
+
+## The encoders in use are:
+
+- libwebp's `cwebp -preset photo -m 6 -z 9 input.png output.webp`
+- libjxl's `cjxl input.png output.jxl "--brotli_effort=11" -e 9`
+- libjxl/jpegli's `cjpegli input.png output.jpeg`
+- libavif's `avifenc -s 0 --ignore-exif --ignore-xmp input.png output.avif`
+
+All encoders are tuned to run at their slowest but highest quality possible for generic images
+to ensure that they can consistently compress into small but usable files.
+
+The pipeline in [scaler.ps1] is currently configured to:
+
+- Emit AVIF at quality 70 with YUV 4:4:4. For most scenes, this generally does well.
+- Emit JPEG at distance 1.5 with YUV 4:4:4. For nearly all scenes, this does well.
+- Emit 360p, 720p and 1080p scales of input.
+  - This allows for data saving and responsiveness as the client doesn't need to download as large of files.
+
+With these quality considerations, for noisy shader-intensive images,
+the recommended encoding settings are of the following:
+
+- AVIF quality 60-80 with YUV 4:4:4. Highly scene dependent.
+- JPEG using Jpegli on distance 2-1.5 with YUV 4:4:4. Hardly changes with scene complexity.
+  - JPEG XL (a.k.a., JXL) should generally perform similary but with marginally smaller file sizes.
+- WebP quality 90 using photo mode.
+  - Important: WebP 90 may not always be within quality constraints and thus is disqualified.
+  - Note: This is often larger than Jpegli.
+
+For flat images, the recommended settings are of the following:
+
+- AVIF quality 50-60 with YUV 4:4:4 or YUV 4:2:0.
+- JPEG using Jpegli on distance 1.5-1 with YUV 4:4:4.
+  - Note: This may be larger than WebP, and introduce more noise than favourable.
+- WebP quality 90 using photo mode.
+  - Being a video codec, this ends up excelling over JPEG.
+
+### Encoders to avoid:
+
+- `libjpeg-turbo`, `mozjpeg`
+  - Offers subpar quality at a given size in comparison to `jpegli`.
+- Non-AOM AVIF encoders, including the GNU Image Manipulation Program
+  - Fails to return the same visual quality at a given size; may introduce artifacts.
+- `cwebp` at quality <80
+  - Always fails in retaining visual quality even in simple scenes.
+
+## Encoded Quality Settings Picked
```

Comment:
This section probably should be renamed as this isn't relevant to the current pictures.